db/schema.rb

```diff
@@ -76,6 +76,17 @@
 
   add_index "linksets", ["rubygem_id"], name: "index_linksets_on_rubygem_id", using: :btree
 
+  create_table "log_tickets", force: :cascade do |t|
+    t.string   "key"
+    t.string   "directory"
```

Comment:
IMHO, calling S3 buckets `directories` is confusing b/c `key` could contain slashes.

If you get rid of the pluggable backends and use S3 stubbing instead, I recommend renaming this to `bucket` to more clearly map to the S3 domain model.